### PR TITLE
[SALAD-23181] WebApp: Redirect to demand alerts on get notified click

### PR DESCRIPTION
**Pull request by vitto-moz**
Based on [this PR ](https://github.com/SaladTechnologies/salad-applications/pull/1232)

### CODE REVIEW

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorPage.tsx [hunk 1]
```diff
@@ -92,6 +96,17 @@ const _DemandMonitorPage: FunctionComponent<Props> = ({
...
+    fetchDemandedHardwarePerformanceList()
+    updateTimerRef.current = setInterval(fetchDemandedHardwarePerformanceList, oneHourInMilliseconds)
+
+    return () => {
+      if (updateTimerRef.current) {
+        clearInterval(updateTimerRef.current)
+      }
+    }
+  }, [fetchDemandedHardwarePerformanceList])
+
```
Comment (rners01) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorPage.tsx:109]: duplicate?

**Review by Maks19 — APPROVED** [review]

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorPage.tsx (hunk 1 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorPage.tsx:109]: @rners01 
removed, 
thanks

Comment (rners01) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorPage.tsx:109]: still there
image

Comment (vitto-moz) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorPage.tsx]: updated

**Review by rners01 — APPROVED** [review]

**Review by Maks19 — APPROVED** [review]

**Pull request merged**